L1Trigger/TrackerTFP/interface/SeedFilter.h

```diff
@@ -0,0 +1,57 @@
+#ifndef L1Trigger_TrackerTFP_SeedFilter_h
+#define L1Trigger_TrackerTFP_SeedFilter_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+#include "L1Trigger/TrackerTFP/interface/LayerEncoding.h"
+
+#include <vector>
+#include <deque>
+
+namespace trackerTFP {
+
+  // Class to find in a region finer rough candidates in r-z
```

Comment:
Clarify comment. e.g. "Class to clean up MHT track candidates with rough tracking in r-z" and state in 1 sentence what r-z tracking entails.

Reply:
what do you mean with what r-z tracking entails

Reply:
By "what r-z tracking entails", I mean give a rough idea what the algo is. e.g. "Find track seed from stubs in two layers, and extrapolate to other layers looking for consistent stubs."